ush/ufoeval/run_ufo_hofx_test.sh

```diff
@@ -11,10 +11,10 @@
 #-------------------------------------------------------------
 #--------------- User modified options below -----------------
 cycle=2021080100
-obtype=amsua_n19
-GDASApp=/work2/noaa/da/cmartin/GDASApp/dev/GDASApp # Change this to your own branch
-workdir=/work2/noaa/da/$LOGNAME/ufoeval/$cycle/$obtype
-yamlpath=$GDASApp/parm/atm/obs/testing/amsua_n19.yaml
+obtype=sondes
```

Comment:
Please 'uncommit' this file, otherwise it looks good

Reply:
@CoryMartin-NOAA done, thanks.